### PR TITLE
Transformers agents cookbook

### PR DESCRIPTION
**Pull request by aymeric-roucher**
# What does this PR do?

Add a cookbook to demonstrate how to use Transformers Agents!

### CODE REVIEW

File: notebooks/en/agents.ipynb [hunk 1]
```diff
@@ -0,0 +1,372 @@
+{
```
Comment (merveenoyan) [inline notebooks/en/agents.ipynb:1]: <p>&gt;maybe give a link to transformers agents</p><p>&gt;enable it: enable the system </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (merveenoyan) [inline notebooks/en/agents.ipynb:1]: <p>&gt;when</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by merveenoyan — APPROVED** [review]
mostly nits, otherwise LGTM

File: notebooks/en/agents.ipynb (hunk 1 above)
Comment (aymeric-roucher) [reply notebooks/en/agents.ipynb:1]: Thank you, added the link and your wording correction!

**Pull request merged**